pkg/debug/Dockerfile

```diff
@@ -63,13 +63,36 @@ ADD https://github.com/pixel/hexedit/archive/refs/tags/1.5.tar.gz ../1.5.tar.gz
 RUN tar -C .. -xzvf ../1.5.tar.gz
 RUN ./autogen.sh && ./configure && make DESTDIR=/out install
 
+WORKDIR /usr/src
+ADD https://cdn.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-9.8p1.tar.gz /usr/src
+RUN tar xvf openssh-9.8p1.tar.gz
+WORKDIR /usr/src/openssh-9.8p1
+RUN ./configure --prefix=/usr --sysconf=/etc/ssh
```

Comment:
Let's be careful with this update, that's not that simple the build. If you look to openssh server Alpine's package, they have a lot of CVE patches applied and specific options for config: For instance, this is for 3.16: https://git.alpinelinux.org/aports/tree/main/openssh/APKBUILD?id=083fb6b05bc08034b1e9efe572c5b0703f01c241#n106

We need to check if all CVE patches are part of this version.... maybe would be better to adapt and build the alpine package instead of manually build from mainstream sources....

Reply:
You're saying that openssl is not fixing their CVEs in the newest version but let the distros do that?

Reply:
Let me clarify, I meant other patches as well, not CVEs... and as explained, there are other custom patches and build options, see my comments below...

Reply:
@christoph-zededa , I'm pretty sure everybody here knows the infamous Debian openssh-server patch, but anyways.... I took a quick look on Alpines patches:

This one fixes redefinition warnings during build, it's not strictly needed, but good to have it....
https://git.alpinelinux.org/aports/tree/main/openssh/avoid-redefined-warnings-when-building-with-utmps.patch

This one includes configs from /etc/ssh/ssh_config.d/*.conf, that's not mandatory for us...
https://git.alpinelinux.org/aports/tree/main/openssh/include-config-dir.patch

This one is fixing something and must be evaluated (the #define fix is already integrated, the other one it's not)....
https://git.alpinelinux.org/aports/tree/main/openssh/fix-utmp.patch

I've also commented on two configuration options that you provided here that is not following Alpine's openssh package, so I'd like to confirm if you really want to change them....

Reply:
we use `--disable-utmp` - so these patches are still needed?

Reply:
I added the fix, but also for the declaration of `sa6` - https://github.com/openssh/openssh-portable/commit/02e16ad95fb1f56ab004b01a10aab89f7103c55d#diff-e3531b64748884f6ea667cb686f117fa64378492519304b44b1ccb2e7dcc9cf2R708

Reply:
Thanks @christoph-zededa , LGTM now...